app/uploaders/disability_claim_document_uploader.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DisabilityClaimDocumentUploader < CarrierWave::Uploader::Base
+  MAX_FILE_SIZE = 25.megabytes
+
+  before :store, :validate_file_size
+
+  def initialize(user_uuid, tracked_item_id)
+    super
+    @user_uuid = user_uuid
+    @tracked_item_id = tracked_item_id
+    set_storage_options!
+  end
+
+  def store_dir
+    "disability_claim_documents/#{@user_uuid}/#{@tracked_item_id}"
+  end
+
+  def extension_white_list
+    %w(pdf gif tiff tif jpeg jpg bmp txt)
+  end
+
+  def move_to_cache
+    false
+  end
+
+  private
+
+  def validate_file_size(file)
+    raise CarrierWave::UploadError, 'File size larger than allowed' if file.size > MAX_FILE_SIZE
+  end
+
+  def set_storage_options!
+    if EVSS_AWS_ACCESS_CREDS
+      self.fog_credentials = {
+        provider:              'AWS',
+        aws_access_key_id:     EVSS_AWS_ACCESS_CREDS[:aws_access_key_id],
+        aws_secret_access_key: EVSS_AWS_ACCESS_CREDS[:aws_secret_access_key],
```

Comment:
since these will need to be stored in credstash most likely, i would just double check to make sure credstash supports hashes and doesn't require strings only.